Jenkinsfile

```diff
@@ -0,0 +1,19 @@
+pipeline {
+    agent {label 'docker'}
+    options {
+        timeout(time: 1, unit: 'HOURS')
```

Comment:
does the timeout has to be 1 hour?
I hope the tests run through way under 5 minutes.

Reply:
Good point, let's set it to 10 minutes first.